templates/500.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+
+{% block content %}
+{{ super() }}
+<div class="container">
+    <div class="mt-10"></div>
+    <h1>Error 500 - Internal server error</h1>
+    <p>You can:
+        <ul>
+            <li>Try again later</li>
+            <li><a href="#" id="playPong">Play pong</a></li>
+            <li>Submit an issue on <a href="https://github.com/saurabhshri/ccextractor-web/issues">GitHub</a></li>
+        </ul>
+    </p>
+    <canvas width="1000" height="500" id="pongCanvas">Your browser doesn't support html5 canvas.</canvas>
+</div>
+
+{% endblock %}
+
```

Comment:
Please make use of layout you're fetching in the handler file. Currently the sidebar has no entries.